server/src/main/java/org/elasticsearch/indices/recovery/RecoveryState.java

```diff
@@ -347,12 +351,22 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         static final String TARGET_THROTTLE_TIME_IN_MILLIS = "target_throttle_time_in_millis";
     }
 
-    public static class Timer implements Streamable {
+    public static class Timer implements Streamable, Writeable {
```

Comment:
I think we can remove `Streamable` from `Timer` then remove `readFrom` entirely.